### PR TITLE
Add missing nix files

### PR DESCRIPTION
**Pull request by kayvank**
adding the missing create-script-contex.nix file to get the build pass.

### CODE REVIEW

**Comment by koslambrou** [comment]
Building `marconi` locally, seems like it doesn't build.

We need to:

* remove `plutus-ledger` from `cardano-streaming.cabal`
* remove `plutus-ledger` and `freer-extras` in `cabal.project`

**Comment by kayvank** [comment]
> Building `marconi` locally, seems like it doesn't build.
> 
> We need to:
> 
> * remove `plutus-ledger` from `cardano-streaming.cabal`
> * remove `plutus-ledger` and `freer-extras` in `cabal.project`



> Building `marconi` locally, seems like it doesn't build.
> 
> We need to:
> 
> * remove `plutus-ledger` from `cardano-streaming.cabal`
> * remove `plutus-ledger` and `freer-extras` in `cabal.project`

from nix develop, all men items seem to be working now.
The following commands are also working in my environment: 
```
nix build .#x86_64-linux.marconi.library.marconi-project.hsPkgs.marconi-chain-index.components.library

nix build .#x86_64-linux.marconi.library.marconi-project.hsPkgs.marconi-sidechain.components.library
```

**Comment by koslambrou** [comment]
Did you try `cabal build all`?

**Comment by koslambrou** [comment]
`cabal build all` failed on my computer. Weird. I have to remove the following from `cabal.project`:

```
allow-older:
  -- freer-extras works with lens-5.2 (rather than 5.2.1)
  , freer-extras:lens
  -- freer-extras works with resource-pool-0.3.1.0 (rather than 0.4.0.0)
  , freer-extras:resource-pool
  -- plutus-ledger works with lens-5.2 (rather than 5.2.1)
  , plutus-ledger:lens
```

Also, had to remove the `plutus-ledger` dependency from `cardano-streaming`.

It's weird that CI doesn't detect this error.

**Comment by koslambrou** [comment]
Mmm... why am I seeing only checks for `marconi-chain-index` and not the other components?

**Comment by kayvank** [comment]
> Did you try `cabal build all`?

yes:
``` 
cabal clean && cabal update && cabal build all && cabal test all
```
works locally for me

**Comment by kayvank** [comment]
> `cabal build all` failed on my computer. Weird. I have to remove the following from `cabal.project`:
> 
> ```
> allow-older:
>   -- freer-extras works with lens-5.2 (rather than 5.2.1)
>   , freer-extras:lens
>   -- freer-extras works with resource-pool-0.3.1.0 (rather than 0.4.0.0)
>   , freer-extras:resource-pool
>   -- plutus-ledger works with lens-5.2 (rather than 5.2.1)
>   , plutus-ledger:lens
> ```
> 
> Also, had to remove the `plutus-ledger` dependency from `cardano-streaming`.
> 
> It's weird that CI doesn't detect this error.

I removed them and force push the PR. 
Taking the plutus-ledger dependency requires removing `Cardano-streaming/examples/Example2.hs`

**Comment by kayvank** [comment]
> Mmm... why am I seeing only checks for `marconi-chain-index` and not the other components?

I am looking into this, I'll update the PR once I have more info.

**Comment by koslambrou** [comment]
Ahh, I think I know why... `plutus-ledger` is on CHaP, so it's fetching it from there.

File: __std__/cells/marconi/library/make-marconi-project.nix [hunk 1]
```diff
@@ -54,7 +54,7 @@ let
         ({ pkgs, ... }: lib.mkIf (pkgs.stdenv.hostPlatform != pkgs.stdenv.buildPlatform) {
           packages = {
             # Things that need plutus-tx-plugin
-            freer-extras.package.buildable = false;
+            # freer-extras.package.buildable = false;
```
Comment (koslambrou) [inline __std__/cells/marconi/library/make-marconi-project.nix:57]: Bin.

**Review by koslambrou — APPROVED** [review]

**Comment by koslambrou** [comment]
In the CI, I see `ci/hydra-build:x86_64-linux.devcontainer-push-docker-image`. That should not be here in `marconi` (I think)

**Comment by kayvank** [comment]
> 

I can take that out, however, I thought we might want to keep it for VScode users.  @koslambrou, Would you confirm if you want to take it out?

**Comment by kayvank** [comment]
removed `devcontainer` from `__std__ `and `README`. Once the build completes, I'll merge this PR into main.  Thank you for your help @koslambrou

**Pull request merged**